user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Charts/Line_and_area_chart.md

```diff
@@ -132,6 +132,55 @@ To configure the component:
 
 Query results are supported as a data feed for this component from DataMiner 10.2.9/10.3.0 onwards. To configure the component to use a GQI query as its data feed:
 
+### [From DataMiner 10.3.0 [CU15]/10.4.0 [CU3]/ 10.4.6 onwards](#tab/tabid-1)
+
+<!--RN 39509-->
+
+1. Create a query data feed. See [Creating a GQI query](xref:Creating_GQI_query).
+
+1. In the *Component* > *Layout* tab, configure the following fields in the *Lines* section:
```

Comment:
Configuring the lines should probably happen after configuring the axis? Or is this one put first because it isn't optional if you want to see a line

Reply:
The lines is indeed mentioned first because this isn't an optional step. Renaming and adding axes is optional. Can I leave it like this, or will this cause issues? @WoutMH

Reply:
It should be fine like this, there will always be an X- & minimum 1 Y-axis